cv_next/components/layout/navbar/popup.tsx

```diff
@@ -5,38 +5,89 @@ import closeIcon from "@/public/images/closeIcon.png";
 import { useRef, useEffect } from "react";
 import Link from "next/link";
 import { useSupabase } from "@/hooks/supabase";
+import profileIcon from "@/public/images/profile.png";
+import { useTheme } from "next-themes";
 
 const navLinks = [
   {
     route: "Login",
     path: "/login",
+    req_login: false,
+  },
+  {
+    route: "Feed",
+    path: "/feed",
+    req_login: true,
   },
   {
     route: "Signout",
     path: "/",
+    req_login: true,
   },
 ];
 
 interface PopupProps {
   closeCb: () => void;
+  userData: UserModel | null;
+  updateSignOut: () => void;
 }
 
-export default function Popup({ closeCb }: PopupProps) {
+export default function Popup({
+  closeCb,
+  userData,
+  updateSignOut,
+}: PopupProps) {
   const dialogRef = useRef<HTMLDialogElement>(null);
+  const supabase = useSupabase();
+  const { theme } = useTheme();
+
   useEffect(() => {
     if (dialogRef.current) {
       dialogRef.current.show();
     }
   }, []);
-  const supabase = useSupabase();
 
   const handleSelection = (route: string) => {
     if (route === "Signout") {
       supabase.auth.signOut();
+      updateSignOut();
     }
     closeCb();
   };
 
+  const matchThemePlaceholderImage =
+    theme == "dark" ? { filter: "invert(0)" } : { filter: "invert(1)" };
```

Comment:
Is this our theming? "invert"?
I mean, it's fine, but it's not really "dark" theme..
@PeroG28 any thoughts?

Reply:
I didn't see the image itself so idk what's better, but I believe that either way you can define the theming manually for the image (even if you choose to stick to image instead of react icon), in a way that is more dark-themed than simple color-inversion.

Reply:
switched both to react icons without the useTheme